package.json

```diff
@@ -98,7 +98,6 @@
     "redux-localstorage": "^0.4.1",
     "resin-cli-form": "^1.4.1",
     "resin-cli-visuals": "^1.2.8",
-    "rindle": "^1.3.0",
```

Comment:
lol, I was also ranting about a dependency called `angular-seconds-to-date`, when I discovered it was something I published long ago :P (https://www.npmjs.com/package/angular-seconds-to-date)